admin_guide/pruning_resources.adoc

```diff
@@ -33,10 +33,58 @@ $ oc adm prune <object_type> <options>
 
 This specifies:
 
-- The `<object_type>` to perform the action on, such as `builds`,
+- The `<object_type>` to perform the action on, such as `groups`, `builds`,
 `deployments`, or `images`.
 - The `<options>` supported to prune that object type.
 
+[[pruning-groups]]
+
+== Pruning groups
+
+In order to prune groups records from an external provider, administrators may
+run the following command:
+
+---
+$ oc adm prune groups --sync-config=path/to/sync/config [<options>]
+---
+
+.Prune Groups CLI Configuration Options
+[cols="4,8",options="header"]
+|===
+
+|Options |Description
+
+.^|`--confirm`
+|Indicate that pruning should occur, instead of performing a dry-run.
+
+.^|`--blacklist`
+|Path to the group blacklist file.
+See xref:../install_config/syncing_groups_with_ldap.adoc#overview[Syncing Groups With LDAP]
+for the structure of this file.
```

Comment:
s/structure of this file./blacklist file structure.

I don't actually see the blacklist or whitelist file structure in this link.

Reply:
I know :( once it'll get added there the link will be ok :)

Reply:
Right, but until then, it's wrong. Do you have the format? If so, we should add it before this change is merged.

Reply:
I don't, I'll try to find one.

Reply:
Let's have it as a followup to this one.

Reply:
Will you check? If we don't add the whitelist/blacklist format to the other topic, then we need to modify this one to not claim that the other topic has it.